frida/core.py

```diff
@@ -198,22 +224,46 @@ class ScriptPayloadMessage(TypedDict):
 ScriptDestroyedCallback = Callable[[], None]
 
 
+class RPCException(Exception):
+    """
+    Wraps remote errors from the script RPC
+    """
+
+    def __str__(self) -> str:
+        return str(self.args[2]) if len(self.args) >= 3 else str(self.args[0])
+
+
 class Script:
     def __init__(self, impl: _frida.Script) -> None:
-        self.exports = ScriptExports(self)
+        self.exports_sync = ScriptExports(self)
```

Comment:
Maybe you should rename `ScriptExports` to `ScriptExportsSync` to better match the rest of the names

Reply:
Changed and added an alias to `ScriptExports`